topic-operator/src/main/java/io/strimzi/operator/topic/v2/BatchingTopicController.java

```diff
@@ -62,6 +62,7 @@ public class BatchingTopicController {
 
     static final String FINALIZER = "strimzi.io/topic-operator";
     static final String MANAGED = "strimzi.io/managed";
+    static final String PAUSED = "strimzi.io/pause-reconciliation";
```

Comment:
Should this be in ResourceAnnotations class in `api` module?

Edit: It is there.